modules/terraform-aci-vrf/main.tf

```diff
@@ -164,27 +198,27 @@ resource "aci_rest_managed" "bgpRtTargetP_ipv4" {
 }
 
 resource "aci_rest_managed" "bgpRtTarget_ipv4_import" {
-  count      = var.bgp_ipv4_import_route_target != "" ? 1 : 0
-  dn         = "${aci_rest_managed.bgpRtTargetP_ipv4[0].dn}/rt-[${var.bgp_ipv4_import_route_target}]-import"
+  for_each   = { for item in local.bgp_ipv4_import_route_target_list : item.key => item.value if var.bgp_ipv4_import_route_target != "" }
```

Comment:
as route targets are list of ip, we should be able to do this with toset function and without need to generate local variables, take a look at how contracts are applied under vrf